integration/src/main/java/scripts/testscripts/RemoveUser.java

```diff
@@ -59,6 +72,34 @@ protected void doSetup(List<TestUserSpecification> testUsers, WorkspaceApi owner
         sharedResourceUser.userEmail,
         "The two test users are distinct");
 
+    // Create a group with group email.
+    groupName = "groupremovetest";
+    ApiClient apiClient = new ApiClient();
+
+    assertNotNull(server.samUri);
+
+    GoogleCredentials userCredential =
+        AuthenticationUtils.getDelegatedUserCredential(
+            testUsers.get(1), ClientTestUtils.TEST_USER_SCOPES);
+    userCredential.refreshIfExpired();
+
+    // build the client object
+    apiClient.setBasePath(server.samUri);
+    AccessToken accessToken = AuthenticationUtils.getAccessToken(userCredential);
+    apiClient.setAccessToken(accessToken.getTokenValue());
+
+    groupApi = new GroupApi(apiClient);
+    try {
```

Comment:
Instead of this, you can do:

groupName = TestUtils.appendRandomNumber("groupremovetest");